tests/testthat/helper_pandas.R

```diff
@@ -0,0 +1,19 @@
+# helper that load before tests are run
+
+# determine the package directory path
+package_dir <- gsub("(PythonEmbedInR).*", "\\1", getwd())
```

Comment:
for those of us weak in regexp's, please add a comment explaining what this does, as well as a couple tests that exercise it.